auth-api/src/auth_api/config.py

```diff
@@ -171,6 +171,11 @@ class _Config():  # pylint: disable=too-few-public-methods
     API_GW_EMAIL_SUFFIX = os.getenv('API_GW_EMAIL_SUFFIX', None)
     API_GW_KC_CLIENT_ID_PATTERN = os.getenv('API_GW_KC_CLIENT_ID_PATTERN', 'api-key-account-{account_id}')
 
+    # Type column prefix text in Tasks table - "New Account" for approving affidavit &
+    # "Access request" for Product request
+    NEW_ACCOUNT_STAFF_REVIEW = 'NEW ACCOUNT'
```

Comment:
Should these be hardcoded or put in env config?

Reply:
Refactored them to enums